hack/chart/voyager/README.md

```diff
@@ -0,0 +1,46 @@
+# Voyager
```

Comment:
Also add This file link in user guide. After the install and run process with title `Run with helm`.